git_machete/cli.py

```diff
@@ -365,13 +365,27 @@ def check_required_param(in_args: List[str], allowed_values_string: str) -> str:
             if dest != current_branch:
                 git.checkout(dest)
         elif cmd == "github":
-            github_allowed_subcommands = "anno-prs|create-pr|retarget-pr"
-            param = check_required_param(parse_options(args, "", ["draft"]),
-                                         github_allowed_subcommands)
+            github_allowed_subcommands = "anno-prs|checkout-pr|create-pr|retarget-pr"
+            list_args = parse_options(args) if args[0] == 'checkout-pr' \
+                else check_required_param(parse_options(args, "", ["draft"]), github_allowed_subcommands)
+            if not list_args:
+                raise MacheteException(f"`git machete github` expects argument(s): {github_allowed_subcommands}")
+
+            param = args[0]
             machete_client.read_definition_file()
             if param == "anno-prs":
                 machete_client.sync_annotations_to_github_prs()
+            elif param == "checkout-pr":
+                if len(list_args) == 1:
+                    raise MacheteException(
+                        "Argument to `git machete github checkout-pr` cannot be empty; expected PR number.")
```

Comment:
```
$ git machete github checkout-pr 204

PR number 204 is not found in repository git-machete
```

PR #204 exists but is already closed. Pls open an issue to make this distinction in the error message (not to be done on this PR)